source/exec/cache.d

```diff
@@ -62,37 +64,65 @@ class Cache: IExecProvider
 	}
 }
 
-private uint getSourceCodeHash(string source)
+private size_t getSourceCodeHash(IExecProvider.RunInput input)
 {
-	import std.digest.crc: crc32Of;
-	auto crc = crc32Of(source);
-	return (cast(uint)crc[0])
-		| ((cast(uint)crc[1]) << 8)
-		| ((cast(uint)crc[2]) << 16)
-		| ((cast(uint)crc[3]) << 24);
+	return input.hashOf;
```

Comment:
I'm not entirely sure why @stonemaster used this implementation, when [`hashOf`](https://dlang.org/library/object/hash_of.html) works nicely out of the box too.
My guess: he wanted to get an integer, but CRC didn't return one and he didn't know about `hashOf`...